rules/S2583/metadata.json

```
@@ -3,7 +3,8 @@
   "type": "BUG",
   "code": {
     "impacts": {
-      "RELIABILITY": "MEDIUM"
+      "RELIABILITY": "MEDIUM",
+      "SECURITY": "MEDIUM"
```

> Conditionally executed code should be reachable

```suggestion
      "RELIABILITY": "MEDIUM",
      "SECURITY": "LOW"
```

Depends a lot on the code I guess.